test/e2e/pkg/infra/docker/docker.go

```diff
@@ -78,7 +78,9 @@ services:
     init: true
     ports:
     - 26656
-    - {{ if .ProxyPort }}{{ .ProxyPort }}:{{ end }}26657
+    - {{ if .RPCProxyPort }}{{ .RPCProxyPort }}:{{ end }}26657
+    - {{ if .GRPCProxyPort }}{{ .GRPCProxyPort }}:{{ end }}26670
+    - {{ if .GRPCLegacyPort }}{{ .GRPCLegacyPort }}:{{ end }}26671
```

Comment:
When compared to main, it appears that this port is in use for the gRPC privileged service:

```
- {{ if .GRPCPrivilegedProxyPort }}{{ .GRPCPrivilegedProxyPort }}:{{ end }}26671
```

So we either change this port number here on this PR, or we have to perform this change on future backports.